dasharo-security/tcg-opal-disk-password.robot

```diff
@@ -47,9 +47,9 @@ DMP001.001 TCG OPAL disk password set and check
     Press Key N Times And Enter    2    ${ARROW_DOWN}
     Save Changes And Reset
     @{password}=    Set Variable    1    2    3
-    Type In New Disk Password    @{password}
+    Type In New Disk Password    ${password}
```

Comment:
@JanPrusinowski Likely we need this to be ported to: https://github.com/Dasharo/open-source-firmware-validation/commits/regression-msi/

Reply:
https://github.com/Dasharo/open-source-firmware-validation/commit/edda9f798c950011a1acedf7cc46e905003570c3